.gitignore

```diff
@@ -29,6 +29,12 @@ venv/
 # Tox envs
 .tox
 
+# Build
+build/
```

Comment:
This is being already contributed by https://github.com/foundation-model-stack/fms-hf-tuning/pull/140 

Also we dont want to ignore complete `build` folder here, the files here should be git tracked.

Reply:
I have approved 140 as well and ran the checks, once checks pass , @kmehant feel free to merge 140 as you have write access

Reply:
also thanks for catching this , I missed it :)